ucsm_apis/admin/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+
+__author__ = """Cisco Ucs"""
+__email__ = 'ucs-python@cisco.com'
+__version__ = '0.9.0.0'
```

Comment:
This should be an empty file with just the license info.. version will be at a ucsm_apis level.. It need not be at every sub-module level